report/src/main/webapp/app/services/patterns.service.js

```diff
@@ -151,13 +160,15 @@ define(['angularAMD', 'metadataService', 'metadataAccessService'],
         }
 
         function isCollectorWithPattern(step) {
-          return step && step.comparators && step.pattern;
+          return step && step.comparators &&
+              step.patterns &&
+              step.patterns.length > 0;
```

Comment:
Is it correctly formatted?

Reply:
I reformatted this code with google formatter again, and now it looks different :)